main.go

```diff
@@ -87,7 +92,7 @@ func main() {
 
 		commonRunnerLabels commaSeparatedStringSlice
 	)
-
+	fmt.Println("ARC VERSION:", getVersion("ARC_VERSION", "VERSION NOT SET"))
```

Comment:
Concept is fine, I'd rather this was output as part of the existing initilisation log output here https://github.com/actions-runner-controller/actions-runner-controller/blob/98cefd105611e8eb81b975ba42d7052e9a50941d/main.go#L220 with the key `version`.